src/convert/mod.rs

```diff
@@ -0,0 +1,69 @@
+//! Conversions between AnyLuaValue and structures.
+//! Also see macros.
+
+pub mod serialize;
+pub mod deserialize;
+
+pub use self::serialize::ToTable;
+pub use self::deserialize::
+{FromTable, LuaDecoder, ConverterError, ConvertResult};
+
+// Tests for serialize <-> deserialize compatability
```

Comment:
If this is tests explicitly for this, shouldn't it be in deseralize.rs? Why is it in mod.rs?